web/modules/custom/portland/modules/portland_address_verifier/README.md

```diff
@@ -48,6 +48,11 @@ Some addresses are technically outside of incorporated areas but are related by
 Allowed values: 1|0
 Default value: 0
 
+**require_portland_city_limits**
+When enabled, only allows addresses within Portland city limits.
+Allowed values: 1|0
+Default value: 0
+
```

Comment:
Documentation for new `out_of_bounds_message` prop is missing

Reply:
Added documentation